sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2330,8 +2331,7 @@ def _setup_and_create_job_cmd_on_local_head(
         with tempfile.NamedTemporaryFile('w', prefix='sky_local_app_') as fp:
             fp.write(ray_command)
             fp.flush()
-            run_file = os.path.basename(fp.name)
-            remote_run_file = f'/tmp/{run_file}'
+            remote_run_file = f'/tmp/sky_local_job_{uuid.uuid4().hex[:8]}'
```

Comment:
Would it be better to put the remote files in a subdir while retaining the filename to be `os.path.basename(fp.name)`? E.g.:

```
remote_run_file = f'/tmp/sky_local/{run_file}'
```

It'll be easier to debug in the future since the local tempfile name will be the same as the one uploaded on remote, but reside in a different directory.

Note that we'll need to make sure we run `mkdir -p /tmp/sky_local` before this or figure out a way to do it in `runner.rsync`. wdyt?

Reply:
I do indeed think this is a better decision, thanks for the suggestion! Added the `mkdir -p /tmp/sky_local` too^